src/event.rs

```diff
@@ -143,6 +150,72 @@ impl EventHandler {
                 screen.redraw_top_bar("", 0)?;
                 Ok(())
             }
+            Event::AddServer(name, connection) => {
```

Comment:
To prevent this method from getting longer. Perhaps create a new one eg. "handle_store_events" which handles these events?

Reply:
Agreed